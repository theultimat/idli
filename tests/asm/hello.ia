    .include "wrapper.ia"

main:
    utxb    'H'
    utxb    'e'
    utxb    'l'
    utxb    'l'
    utxb    'o'
    utxb    32      # Assembler has a bug on parsing ' ' so avoid for now...
    utxb    'W'
    utxb    'o'
    utxb    'r'
    utxb    'l'
    utxb    'd'
    utxb    '!'
    ret
