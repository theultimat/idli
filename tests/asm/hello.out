0x6548
0x6c6c
0x206f
0x6f57
0x6c72
0x2164
